src/Bbt.Maybe.Tests/Bbt.Maybe.Tests.csproj

```diff
@@ -21,6 +21,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
+    <PackageReference Include="Shouldly" Version="4.0.0-beta0001" />
```

Comment:
Should needs to be added to code coverage exclusion [here](https://github.com/bbtsoftware/BBT.Maybe/blob/develop/setup.cake#L22).